doc/torque_api.md

```diff
@@ -0,0 +1,167 @@
+# Torque API Methods
```

Comment:
yeah, it's just in one place, `API.md` serves as an index https://github.com/CartoDB/torque/blob/newdocs/doc/API.md :P